### PR TITLE
Update 'headblur' example

### LINKED ISSUE #167: headblur.py: ImportError to_fxfy
First, thanks for the great project!

I use the current codebase from github. when I want to start the headblur.py example, it throws an Input Error:

> $ python headblur.py
> 
> ```
> Traceback (most recent call last):
>   File "headblur.py", line 4, in <module>
>     from moviepy.video.tools.tracking import manual_tracking, to_fxfy
> ImportError: cannot import name to_fxfy
> ```

Please help

### LINKED ISSUE #850: Headblur example
Hi Moviepy Team,

Example code for headblur is very old. Functions like to_fxfy are not present in code now.

Can you please put a full example to blur certain position of a video using moviepy? It will be really helpful for me in my experimentation with moviepy.

Thanks,
Deepak.

### PR DESCRIPTION
**Pull request by mondeja**
+ Headblur example updated with current implementation of `moviepy.video.tools.tracking.manual_tracking`.
+ Included `media/chaplin.mp4` file from ##1155.
+ Fixed errors in `headblur` FX (see #1062).

This pull request closes #1155, closes #749, closes #850, closes #167, closes #1062. Contributes to #338.

- [x] I have formatted my code using `black -t py36`

### CODE REVIEW

File: examples/headblur.py [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
-# LOAD THE TRACKING DATA AND CONVERT IT TO FUNCTIONS x(t),fy(t)
+# LOAD THE TRACKING DATA AND CONVERT IT TO TRAJECTORY INTERPOLATORS x(t), fy(t)
 
-with open("../../chaplin_txy.dat", "r") as file:
-    fx, fy = to_fxfy(pickle.load(file))
+traj = Trajectory.from_file("blurred_trajectory.txt")
 
 
-# BLUR CHAPLIN'S HEAD IN THE CLIP
+# BLUR CHAPLIN'S HEAD IN THE CLIP PASSING xi(t) and yi(t) FUCTIONS
```
Comment (tburrows13) [inline examples/headblur.py:26]: *Functions :)

**Comment by tburrows13** [comment]
Would you mind confirming that #1155 can be closed if this is merged? That PR removes the subclip... do you think that should be done?

**Comment by mondeja** [comment]
> Would you mind confirming that #1155 can be closed if this is merged? That PR removes the subclip... do you think that should be done?

Oh, I haven't notice. It seems that the `.subclip` should be removed. I'll make another test and update it here.

> PS Are you on the Gitter?

Yes, I'm inside.

**Comment by mondeja** [comment]
It seems that I was using the v1.0.3 of moviepy at the time of opening this pull. In the last commit, I've fixed two errors that I've found in `headblur` FX using `opencv-python` version defined at setup (`3.4.13.47`). Now it works perfect.

**Review by tburrows13 — APPROVED** [review]

**Pull request merged**